subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -72,11 +72,26 @@ class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec {
     }
 
     @Issue("https://github.com/gradle/gradle/issues/1365")
-    def "detect changes to broken symlink outputs in #outputType"() {
+    def "detect changes to broken symlink outputs in OutputDirectory"() {
         def root = file("root").createDir()
         def target = file("target")
         def link = root.file("link")
-        buildFile << script(target, link)
+        buildFile << """
+            import java.nio.file.*
+            class ProducesLink extends DefaultTask {
+                @OutputDirectory File outputDirectory
```

Comment:
Any reason you use a `File` here and `Path` in `detect changes to broken symlink outputs in OutputFile`?

Reply:
I'm not referencing it directly here, and need it to be Path in the other. If diversity is not a virtue here, I'll change both to `Path`.

Reply:
Done